lib/spack/spack/detection/path.py

```diff
@@ -44,36 +57,11 @@ def executables_in_path(path_hints=None):
         path_hints (list): list of paths to be searched. If None the list will be
             constructed based on the PATH environment variable.
     """
-    # If we're on a Windows box, run vswhere,
-    # steal the installationPath using windows_os.py logic,
-    # construct paths to CMake and Ninja, add to PATH
     path_hints = path_hints or spack.util.environment.get_path("PATH")
```

Comment:
I intended the remove the `path_hints` defaulting to path from `exectuables_in_path` but must have dropped the change switching branches, would that resolve the issue here? Or are you referring to the default in this method?

Reply:
> I intended the remove the path_hints defaulting to path from exectuables_in_path but must have dropped the change switching branches, would that resolve the issue here?

Yes (once you do that though the test in `test/cmd/external` will have to pass in the path when it calls `executables_in_path` or else it will fail.